types/esbuild-copy-static-files/package.json

```diff
@@ -5,6 +5,10 @@
     "projects": [
         "https://github.com/nickjj/esbuild-copy-static-files"
     ],
+    "dependencies": {
+        "esbuild": "*",
```

Comment:
This would be a pretty unfortunate dep to add in the repo thanks to just how many platform-specific deps this will pull in; is there no way to declare things such that this dependency is not needed?

Reply:
It would be quite unfortunate for DT(-tools) to not accept `esbuild` as a valid dep. 

As of today, `esbuild`  reached 36.9M weekly download. One can imagine there will be more and more unofficial, js-based plugin / package being published along with `esbuild` rapid dev. 

If `esbuild` truly cannot be added in DT typedef, type author in DT or type consumer would be quite restricted in the way they maintain or utilize these types. 
- In existing code, we have `setup(build: any): void;` to make things work. Doing `any` is not too helpful for TS user.
- Alternative approach is to do generic on `setup(build: T): void;` and let consumer to substitute the types, but this may not be very intuitive or handy.

Rather than letting different type authors thinking on their own workarounds, I would like to ask DT to (re-)consider the possibility to support adding `esbuild` as a valid dep. This would make things much easier for different stakeholders.

Reply:
It's not about it being popular, but about it being large. But, I don't think it's actually a big deal... We allow `react-native` as a dep and that's 72MB! So, nevermind.